mappings/net/minecraft/screen/ScreenHandler.mapping

```diff
@@ -10,6 +10,7 @@ CLASS net/minecraft/class_1703 net/minecraft/screen/ScreenHandler
 	FIELD field_30730 EMPTY_SPACE_SLOT_INDEX I
 		COMMENT A special slot index value ({@value}) indicating that the player has clicked outside the main panel
 		COMMENT of a screen. Used for dropping the cursor stack.
+	FIELD field_34024 nonce I
```

Comment:
I’m sure this doesn’t mean what comes to my mind (from the uk). Something else might better here as I don’t know what this word is supposed to mean.

Reply:
It's an integer that is used to validate if something is in sync. Better name maybe? (Implemented as incremental id, but can be any integer as long as it changes per packet)

Reply:
Is this a value to ensure modifications are up-to-date? Think about java collections' `modCount`.

Reply:
`version` might be better; we used `version` back when vanilla had a multithreading-error-prone tag container implementation that used an int field to check if a tag reference is up-to-date.

Reply:
Even `revision` could work (IMO sounds more automated than `version` if you think about software dev)